tasks/install-with-snap.yml

```diff
@@ -1,41 +1,36 @@
 ---
 - name: Ensure snapd is installed.
-  package:
+  ansible.builtin.package:
     name: snapd
     state: present
-  register: snapd_install
+  notify:
+    - Update snap after install
 
 - name: Ensure snapd is enabled.
-  systemd:
+  ansible.builtin.systemd:
     name: snapd.socket
     enabled: true
     state: started
 
 - name: Enable classic snap support.
-  file:
+  ansible.builtin.file:
     src: /var/lib/snapd/snap
     dest: /snap
     state: link
   when: ansible_os_family != "Debian"
 
-- name: Update snap after install.
```

Comment:
Changing this could change the behavior of this role...

Reply:
Ansible-lint gives a reasonable failure about this.

```
no-handler: Tasks that run when changed should likely be handlers.
tasks/install-with-snap.yml:21 Task/Handler: Update snap after install.
```
Update action was moved into handlers

[ansible-lint-role-certbot.txt](https://github.com/user-attachments/files/16438012/ansible-lint-role-certbot.txt)

Reply:
Ehm, sorry, stupid 'git add' ))